### ISSUE TITLE
NDNP batch ingest refinements

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#154]
_Placeholder ticket to enumerate miscellaneous medium-priority improvements that should be made to NDNP ingest functionality._

**1. NewspaperTitle title doesn't exactly match child Issue**

NewspaperTitle: `The Park Record`
 
NewspaperIssue: `The Park Record (Park City, UT)`

**2. Issue title syntax and date formatting**

This: `The Park Record (Park City, UT) (1935-08-02)`
  
Should be: `The Park Record (Park City, UT): August 2, 1935`

**3. Issue edition value**

**Removed from scope: moved to #160.**

This should come from `//MODS:detail[@type='edition']/MODS:number`.
  
(In `batch_local` ingest, this was set as the value of `//MODS:detail[@type='edition']/MODS:caption`)

**4. Page title syntax**

This: `The Park Record (Park City, UT) (1935-08-02): Page 1`
  
Should be: `The Park Record (Park City, UT): August 2, 1935: Page 1`

**5. Page identifier**

Attempt to just use filename without directory and extension.
  
_batch_local_:
  
This: `/mnt/nash.iarchives.com/data01/jobq/root/projects/production/LeanProcessing/UofU/Park_Record/Park_Record_Set01/tpr_19350705-19380630/ocr/0657b.tif`
    
Should be: `0657b`
  
_batch_ver_01_:

This: `././0006.tif`
    
Should be: `0006`

**Comment by seanupton** [comment]
Quick note: be58060bb989a6553436c2cb70d8935d82a0d41a implements 1, 2, and 4 in the above description — however, to be consistent with the publication title, place-of-publication label is omitted (no parenthetical) in the issue and page titles in this implementation.  I believe this consistency across instances is desirable, but I want to make sure this implementation decision makes sense?

This will look like (in order, publication, issue, page titles):
```
ACME Falls Times
ACME Falls Times: December 7, 1941
ACME Falls Times: December 7, 1941: Page 2
```

**Comment by seanupton** [comment]
Note: fffdbf676277e80c10b8db6232838c89df659f3d implements changes required for item 5.

**Comment by seanupton** [comment]
Punting item 3 on this list to issue #160.

**Comment by seanupton** [comment]
Closed, merged to master via PR #164.
